src/include/dav1d/headers.rs

```diff
@@ -378,12 +378,17 @@ impl From<Rav1dWarpedMotionParams> for Dav1dWarpedMotionParams {
 }
 
 // TODO(kkysen) Eventually the [`impl Default`] might not be needed.
-#[derive(Clone, Copy, PartialEq, Eq, EnumCount, FromRepr, Default)]
+/// Pixel layout of a frame.
+#[derive(Clone, Copy, PartialEq, Eq, EnumCount, FromRepr, Default, Debug)]
 pub enum Rav1dPixelLayout {
+    /// Monochrome.
     #[default]
     I400 = 0,
+    /// 4:2:0 planar.
     I420 = 1,
+    /// 4:2:2 planar.
     I422 = 2,
+    /// 4:4:4 planar.
     I444 = 3,
 }
```

Comment:
These kinds of docs are generally very useful, so if you'd like to pull them out into a separate PR, I can review and merge that right away.